relayer/cmd/root.go

```diff
@@ -24,6 +24,12 @@ var rootCmd = &cobra.Command{
 func init() {
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVar(&configFile, "config", "", "config file")
+	rootCmd.PersistentFlags().Int(
```

Comment:
Since these `direction` and ` headers` flags are only meaningful for the `run` subcommand, can we move them over to `cmd/run.go`?

I left the `config` flag on the root command just in case we add more subcommands that rely on a common config.